content/en/about-structure/index.md

```diff
@@ -422,29 +422,35 @@ sections:
       columns: '1'
       css_class: d-flex align-items-center
       spacing:
-        padding: ['20px', '0', '20px', '0']
+        padding: ['20px', '0', '40px', '0']
 
-  - block: hero
+  - block: markdown
     content:
       title:
-      image:
-        # Reference an image in your `assets/media/` folder
-        filename:
-      cta:
-        url: 'https://eu1.hubs.ly/H06kZ0d0'
-        label: Interested in investing?
-        icon_pack: fas
-        icon: euro-sign
-      cta_alt:
-        url: 'mailto: neil@alkem.io'
-        label: Do well and do good
+      text: |-
+        <div style="display: flex; justify-content: center; align-items: center;">
+          <!-- CTA Button -->
+          <a href="./documents/alkemio_investor_deck_08_2024.pdf" 
```

Comment:
Not via a hubspot form deliberately?

Reply:
I figured that since all other documents are stored in the documents folder of the page, this would be consistent with the rest. But I can do it via hubspot too, if it's preferable!

Reply:
It is better for this document to be via hubspot, it is promotional in nature as opposed to being transparent - so I would like it via Hubspot...

Reply:
@techsmyth changed to a hubspot link!